pvlib/irradiance.py

```diff
@@ -321,9 +325,9 @@ def get_total_irradiance(surface_tilt, surface_azimuth,
 
     Notes
     -----
-    Models ``'haydavies'``, ``'reindl'``, ``'perez'`` and ``'perez-driesse'``
-    require ``'dni_extra'``. Values can be calculated using
-    :py:func:`~pvlib.irradiance.get_extra_radiation`.
+    Models ``'haydavies'``, ``'reindl'``, ``'perez'``, ``'perez-driesse'``,
+    and ``'muneer'`` require ``'dni_extra'``. Values can be calculated
```

Comment:
possibly enhanced clarity, or possibly just a waste of words:
```suggestion
    and ``'muneer'`` require ``'dni_extra'``. Values of ``'dni_extra'`` can be calculated
```